containers/Loan/Issue/index.tsx

```diff
@@ -52,7 +52,7 @@ class IssueLoan extends React.Component<Props, State> {
 
   onRegistryAddressValueChange = async (event: React.ChangeEvent<HTMLInputElement>) => {
     const currentRegistryAddress = event.currentTarget.value;
-    this.setState({
+    await this.setState({
```

Comment:
setState is not an async function, right?

Reply:
I think it is async 🙈

Reply:
or at least there is no gurantee that the state is updated immediately